bundles/org.openhab.binding.mybmw/src/main/resources/OH-INF/i18n/mybmw.properties

```diff
@@ -78,6 +78,7 @@ channel-type.mybmw.check-control-channel.label = Check Control
 channel-type.mybmw.checkcontrol-details-channel.label = CheckControl Details
 channel-type.mybmw.checkcontrol-name-channel.label = CheckControl Description
 channel-type.mybmw.checkcontrol-severity-channel.label = Severity Level
+channel-type.mybmw.home-distance-channel.label = Distance from Home
```

Comment:
You need to update this file, as it only contains label, not description.

Reply:
Ah, got it! Update and pushed